examples/k8s-sidecar/proxy_with_iam_workload_identity.yaml

```diff
@@ -0,0 +1,83 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloud_sql_proxy_k8s_sa]
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: <YOUR-DEPLOYMENT-NAME>
+spec:
+  selector:
+    matchLabels:
+      app: <YOUR-APPLICATION-NAME>
+  template:
+    metadata:
+      labels:
+        app: <YOUR-APPLICATION-NAME>
+    spec:
+      serviceAccountName: <YOUR-KSA-NAME>
+      # [END cloud_sql_proxy_k8s_sa]
+      # [START cloud_sql_proxy_k8s_secrets]
+      containers:
+      - name: <YOUR-APPLICATION-NAME>
+        # ... other container configuration
+        env:
+        - name: DB_USER
+          # For a service account, this is the service account's email
+          # without the .gserviceaccount.com domain suffix.
+          # Example: gke-service-account@my-project.iam
+          value: <YOUR-CLOUD-SQL-IAM-DB-USER>
+        - name: DB_NAME
+          value: <YOUR-DB-NAME>
+      # [END cloud_sql_proxy_k8s_secrets]
+      # [START cloud_sql_proxy_k8s_container]
+      - name: cloud-sql-proxy
+        # It is recommended to use the latest version of the Cloud SQL proxy
+        # Make sure to update on a regular schedule!
+        image: gcr.io/cloud-sql-connectors/cloud-sql-proxy:2.0.0.preview.0  # make sure the use the latest version
+        args:
+          # If connecting from a VPC-native GKE cluster, you can use the
+          # following flag to have the proxy connect over private IP
+          # - "--private-ip"
+
+          # Enable Automatic IAM Authentication
+          - "--enable-iam-login" # NOT SURE IF THIS IS THE CORRECT ARGUMENT!
```

Comment:
I'm not sure if this is the correct argument. I'm not sure how to find the arguments list. We've been using the command `- "-enable_iam_login"`, but I'm not sure how that translates to an argument. The [Quickstart](https://cloud.google.com/sql/docs/mysql/connect-instance-kubernetes) uses commands, but the [Connect From](https://cloud.google.com/sql/docs/mysql/connect-instance-kubernetes) documentation uses args. I'm not certain how to translate from one to the other.